src/txmempool.cpp

```diff
@@ -890,11 +889,11 @@ const CTransaction* CTxMemPool::GetConflictTx(const COutPoint& prevout) const
     return it == mapNextTx.end() ? nullptr : it->second;
 }
 
-Optional<CTxMemPool::txiter> CTxMemPool::GetIter(const uint256& txid) const
+std::optional<CTxMemPool::txiter> CTxMemPool::GetIter(const uint256& txid) const
 {
     auto it = mapTx.find(txid);
     if (it != mapTx.end()) return it;
-    return Optional<txiter>{};
+    return std::optional<txiter>{};
```

Comment:
```suggestion
    return {};
```

Reply:
This should also finally quell the `-Wmaybe-uninitialized ` false positives we've seen. i.e #21318, #21248.